subsys/net/ip/net_pkt.c

```diff
@@ -1848,7 +1848,7 @@ int net_pkt_pull(struct net_pkt *pkt, size_t length)
 	net_pkt_cursor_backup(pkt, &backup);
```

Comment:
No need to duplicate the "net" prefix so many times in commit subject, for the first commit you can just say `net: pkt: Fixed packet buffer corruption in net_pkt_pull()`